openmp/runtime/test/ompt/tasks/explicit_task_thread_num.c

```diff
@@ -59,10 +59,8 @@ int main()
   // CHECK-DAG: {{^}}[[WORKER_ID:[0-9]+]]: ompt_event_implicit_task_begin
```

Comment:
I would prefer a different fix: Remove all the MASTER_ID lines, change this line:
```suggestion
  // CHECK: {{^}}[[WORKER_ID:[0-9]+]]: ompt_event_implicit_task_begin{{.*}}thread_num=1
```
And revert the changes below. 

If the line becomes to long for clang-format, moving the CHECK lines to the end of the file will safe 2 trailing white spaces, `WORKER_ID`-> `WID` will safe another 6 characters, or change ` ompt_event_` -> `{{.*}}`

Reply:
Thanks for your review and advice, I've fixed it already.